packages/cli/src/args.ts

```diff
@@ -13,6 +13,7 @@ export const help = () => `
       deploy               [path]      Performs a deployment ${chalk.bold(
         '(default)'
       )}
+      build                            Build the project locally
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** 

```suggestion
      build                            Build the project locally into `./vercel/output`
```